toast-tk-runtime/src/main/java/io/toast/tk/runtime/report/XmlPrettierHelper.java

```diff
@@ -0,0 +1,104 @@
+package io.toast.tk.runtime.report;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class XmlPrettierHelper {
+
+	static String getPrettySentenceText(String str) {
+		if(str.contains("*")) {
+			String separator = "\\*";
+			String[] lines = str.split(separator);
+			List<String> res = getPrettyXmlText(lines);
+			String resultat = String.join("*", res);
+			if(lines.length % 2  == 0) {
+				resultat += "*";
+			}
+			return resultat;
+		} else {
+			return getPrettyXmlText(str);
+		}
+	}
+	static List<String> getPrettyXmlText(String[] lines) {
+		List<String> res = new ArrayList<String>();
+		for(String line : lines) {
+			if((line.indexOf("<") < line.indexOf(">") + 1) 
+				&& line.contains("<") && line.contains(">") ) {
+				res.add(System.lineSeparator() + getPrettyXmlText(line)
+						+ System.lineSeparator());
+			} else {
+				if(!"".equals(line)) {
+					res.add(line);
+				}
+			}
+		}
+		return res;
+	}
+	static String getPrettyXmlText(String str) {
+		// IF this is not an XML
+		if(!(str.indexOf("<") < str.indexOf(">") + 1)) {
+			return str;
+		}
+		
+		String separator = System.lineSeparator();
+		String[] lines = str.split(separator);
+		List<String> res = new ArrayList<String>();
+		int tabNb = 0;
+		for(String line : lines) {
+			boolean isInside = true;
+			boolean lastIsInside = true;
+			boolean asToClose = false;
+			if(line.contains("<") && line.contains(">")) {
+				String lineTemp = line.trim()
+						.replace("\t", "")
+						.replace("\n", "")
+						.replace("\r", "");
+				int index = lineTemp.indexOf("<");
+				String firstPart = lineTemp.substring(0, index).trim();
+				if(!"".equals(firstPart.trim())) {
+					res.add(firstPart);
+				}
+				
+				String secondPart = lineTemp.substring(index);
+				for(String lineSplit : secondPart.split("<|>")) {
+					String lineResult = "";
+					lineSplit = lineSplit.trim();
+					if(lineSplit.equals("")) {
+						isInside = true;
+						lastIsInside = isInside;
+						continue;
+					} else {
+						isInside = lastIsInside ? false : true;
+					}
+					if(lineSplit.startsWith("/")) {
+						tabNb += -1;
+					}
+					for(int i = 1; i <= tabNb; i++) {
+						lineResult = "\t" + lineResult;
+					}
+					if(!lineSplit.startsWith("/") & !lineSplit.endsWith("/") 
+							& !lineSplit.startsWith("?") & !lineSplit.endsWith("?")
+							& lastIsInside) {
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Correct this "&" to "&&". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2178)